src/app/modules/workspace/__tests__/workspace.routes.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { ObjectId } from 'bson-ext'
+import mongoose from 'mongoose'
+import supertest, { Session } from 'supertest-session'
+
+import { getWorkspaceModel } from 'src/app/models/workspace.server.model'
+import { WorkspacesRouter } from 'src/app/routes/api/v3/admin/workspaces'
+
+import {
+  createAuthedSession,
+  logoutSession,
+} from 'tests/integration/helpers/express-auth'
+import { setupApp } from 'tests/integration/helpers/express-setup'
+import dbHandler from 'tests/unit/backend/helpers/jest-db'
+import { jsonParseStringify } from 'tests/unit/backend/helpers/serialize-data'
+
+const WorkspaceModel = getWorkspaceModel(mongoose)
+
+const app = setupApp('/workspaces', WorkspacesRouter, {
+  setupWithAuth: true,
+})
+
+const MOCK_USER_ID = new ObjectId()
+const MOCK_FORM_ID = new ObjectId()
+const MOCK_WORKSPACE_ID = new ObjectId()
+const MOCK_WORKSPACE_FIELDS = {
```

Comment:
nit: should this be called `MOCK_WORKSPACE_DOC` instead?

Reply:
Yes it should, thanks